### PR TITLE
Added the webpack development server

### PR DESCRIPTION
**Pull request by nick-bc-dev**
I added the development server.

### CODE REVIEW

**Comment by willeastcott** [comment]
Thanks for this! 😄 Can you provide a bit more information in the description about what this PR does, why it's useful and how to use it please?

**Comment by nick-bc-dev** [comment]
Thank you for your comment. I was looking for the webpack template for playcanvas and reached this project.
It was great, I could build my project and serve the build using this template. But I found that the live reloading did not work. So I tried to fix it on my own. 
Regards,

File: package.json [hunk 1]
```diff
@@ -19,16 +19,30 @@
...
   },
+  "type": "module",
   "devDependencies": {
+    "@babel/core": "^7.17.8",
+    "@babel/preset-env": "^7.16.11",
+    "babel-loader": "^8.2.4",
+    "clean-webpack-plugin": "^4.0.0",
+    "copy-webpack-plugin": "^10.2.4",
     "css-loader": "^6.5.1",
+    "dat.gui": "^0.7.9",
```
Comment (willeastcott) [inline package.json]: Is this really required for a minimal template? Same question for the other packages added here. Are there any others that can be considered unnecessary?

Comment (nick-bc-dev) [reply package.json]: dat.gui is not needed for a minimal template, and I will check again. I didn't take care of software quality in production.

Comment (kgc0108) [reply package.json:33]: Do I need to close this PR and open a new one?

**Comment by nick-bc-dev** [comment]
I added minimal packages and settings for the webpack development server. Please have a look.

**Review by willeastcott — APPROVED** [review]

**Comment by willeastcott** [comment]
Great, thanks!

**Pull request merged**